mne/realtime/epochs.py

```diff
@@ -205,9 +215,82 @@ def __init__(self, client, event_id, tmin, tmax, stim_channel='STI 014',
 
     @property
     def events(self):
-        """The events associated with the epochs currently in the queue."""
+        """
+        Returns the events associated with the epochs currently in the queue.
+
+        Returns
+        -------
+        array of int, shape (n_events, 3)
+        event array
+        """
+        """"""
         return np.array(self._events)
 
+    @events.setter
+    def events(self, new_events):
+        """
+        Updates the internal event list.
+
+        Parameters
+        ----------
+        new_events : array of int, shape (n_events, 3)
+            new events
+        """
+        self._events = [tuple(new_events[i, :])
+                        for i in range(len(new_events))]
+
+    @property
+    def selection(self):
+        """
+        Returns the current epoch selection.
+        Returns
```

Comment:
This docstring is not formatted properly. Better to just say `"""Array of integers of the current selection."""`